packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -506,9 +506,12 @@ export class PipelineEditor extends React.Component<
   }
 
   propertiesActionHandler(id: string, appData: any, data: any): void {
+    const propertyId = { name: data.parameter_ref };
+    const filename = this.propertiesController.getPropertyValue('filename');
```

Comment:
Once #861 is merged this will need to be updated to
```suggestion
    const filename = PipelineService.getWorkspaceRelativeNodePath(
       this.widgetContext.path,
       this.propertiesController.getPropertyValue('filename')
     );
```

Reply:
done